### DISCUSSION TITLE
New Link for running windows and macos and linux

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#934] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
its http://computernewb.com/collab-vm and http://computernewb.com/collab-vm/user-vm
NOTE: can be controlled itself because its public because it is vnc servers
DO NOT LOGIN in everywhere
NOTE: half of the VMs is nsfw
and anyone can see you 
and add that in readme

**Comment by ghost** [comment]
i found extra link for linux it is http://onworks.net and http://distrosea.net

**Reply by Braden000** [reply]
there is a way to get past go guardian kinda but you need to find a password just copy the blocked link and delete the current link then paste the link 2 times
this
https://blocked.goguardian.com/?ctx=YWZpPTMxMDYzMDYmYXBpPTQ3MzAwJm9pPTE0MDc0MDYzODAwJm91PWNocm9tZSUzQSUyRiUyRmluc3BlY3QlMkYmcnM9QURNSU5fU0lURV9GSUxURVImc3Q9Y2hyb21pdW0tbSZ2PTE%3D&sum=6252b66dhttps://blocked.goguardian.com/?ctx=YWZpPTMxMDYzMDYmYXBpPTQ3MzAwJm9pPTE0MDc0MDYzODAwJm91PWNocm9tZSUzQSUyRiUyRmluc3BlY3QlMkYmcnM9QURNSU5fU0lURV9GSUxURVImc3Q9Y2hyb21pdW0tbSZ2PTE%3D&sum=6252b66d

not this 
https://blocked.goguardian.com/?ctx=YWZpPTMxMDYzMDYmYXBpPTQ3MzAwJm9pPTE0MDc0MDYzODAwJm91PWNocm9tZSUzQSUyRiUyRmluc3BlY3QlMkYmcnM9QURNSU5fU0lURV9GSUxURVImc3Q9Y2hyb21pdW0tbSZ2PTE%3D&sum=6252b66d

**Reply by EnderBeastyt** [reply]
then what do i do

**Comment by ghost** [comment]
half of the vms has blocked websites in computernewb
and read the rules and faq

**Comment by ghost** [comment]
also see http://computernewb.com

**Comment by ghost** [comment]
macmac means macos

**Comment by ghost** [comment]
EVEN I FOUND NEW LINK (NEEDS GITHUB CODESPACES OR GITPOD)
instructions is https://git.mollomm1.dev/Mollomm1/DesktopOnCodespaces

**Comment by ghost** [comment]
plus follow this youtuber https://youtube.com/@TigerFishy for proxies

**Comment by ghost** [comment]
NEW LINK FOR OLD OSES is http://copy.sh/v86 and http://pcjs.org AND THE MERCURYWORKSHOP HAS ANURAOS link is http://anura.mercurywork.shop and anura.pro

**Comment by ghost** [comment]
AND YOU CAN HAVE WINDOWS it is browserling.com and lamdatest.net and apponfly.com

**Comment by funkymokey** [comment]
How many edibles did you eat to be beyond high

**Reply by CGRmags** [reply]
45

**Reply by ghost** [reply]
it lets you control collaborative virtual machines

**Reply by Charlieis1nsan3** [reply]
im surprised u know what that is
im ate 10000000
i eat carts too

**Reply by Charlieis1nsan3** [reply]
im hit dat shit

**Reply by funkymokey** [reply]
are you high
